src/elements/ChannelIcon/index.js

```diff
@@ -7,41 +7,56 @@ import E2GR from '../../assets/channels/E2GR.svg';
 import E2NO from '../../assets/channels/E2NO.svg';
 import E2RUG from '../../assets/channels/E2RUG.svg';
 
-export const iconMap = {
-  E: {
+const getChannelIcon = callsign => {
```

Comment:
I'm thinking something like this would actually read better:
```suggestion
import * as channels from '../../assets/channels';

const baseChannel = callSign => callSign.match(/E\d?/)[0];

const getChannelIcon = callSign => channels[callSign] || channels[baseChannel(callSign)] || channels.E;
```

Reply:
And I'll move all the data into `../../assets/channels/index.js`

like this:
```js
import e from './E.svg';
import e1 from './E1.svg';
import e2 from './E2.svg';
import e2GR from './E2GR.svg';
import e2NO from './E2NO.svg';
import e2RUG from './E2RUG.svg';

export const E = {
  src: e,
  altText: 'Eurosport',
  widthRatio: 0.87,
};

export const E1 = {
  src: e1,
  altText: 'Eurosport 1',
  widthRatio: 1.42,
};

export const E2 = {
  src: e2,
  altText: 'Eurosport 2',
  widthRatio: 1.66,
};

export const E2GR = {
  src: e2GR,
  altText: 'Eurosport 2',
  widthRatio: 3.5,
};

export const E2NO = {
  src: e2NO,
  altText: 'Eurosport 2',
  widthRatio: 2.06,
};

export const E2RUG = {
  src: e2RUG,
  altText: 'Eurosport 2',
  widthRatio: 4,
};
```